src/styles/getMuiTheme.js

```diff
@@ -120,6 +126,10 @@ export default function getMuiTheme(baseTheme, muiTheme) {
     listItem: {
       nestedLevelDepth: 18,
       secondaryTextColor: Colors.lightBlack,
+      leftIconColor: Colors.grey600,
+      leftIconFill: Colors.grey600,
+      rightIconColor: Colors.grey400,
+      rightIconFill: Colors.grey400,
```

Comment:
I think it's good to put fill into muiTheme since we are migrating color from every component